components/ButtonBar/src/Theming/MDCButtonBar+Theming.h

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialButtonBar.h"
+#import "MaterialContainerScheme.h"
+
+@interface MDCButtonBar (MaterialTheming)
```

Comment:
Should we document the category?

Reply:
We don't because header docs don't get picked up for categories.

Reply:
Makes sense. Would you pease add in-line comments for the category, then? I think adding the extra `*` wouldn't hurt, but if you prefer not to use it then normal block comments are OK.

Reply:
Done.